ibis/expr/types/relations.py

```diff
@@ -2401,6 +2403,28 @@ def drop(self, *fields: str | Selector) -> Table:
         │ Torgersen │          19.3 │               193 │        3450 │ female │ … │
         └───────────┴───────────────┴───────────────────┴─────────────┴────────┴───┘
 
+        If a given column does not exist, then by default an error is raised.
+        If you want to ignore missing columns, you can pass `missing_ok=True`.
+        This is useful if you need a column to not exist,
+        but you don't know if it does or not.
+
+        >>> t.drop("column_that_must_not_exist_afterwards")  # doctest: +SKIP
```

Comment:
I left out the specific error because I am worried that is going to change and become stale.
I would only want to show the actual error type if we remove the `doctest: +SKIP` marker, but I thought the extra boilerplate to actually be able to catch the error was not worth it. Open to adjusting this though.